### PR TITLE
base navigation bottom created, layoutFixedScrollable component updat…

### PR DESCRIPTION
**Pull request by abdullahcalisir12**
…ed according to Volkswagen project

# Issue Being Addressed
#191 
Put the issue(s) # that your PR is addressing here. e.g. #15, #29

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ x] Refactor
[ x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For Refactors:** What are we refactoring and why? How do your changes address this need for refactoring?

I refactored tailwind.css to use same classes at many components. I added new spacing to tailwind-config for fab button.

**For New Features:** What feature are we adding? Explain your technical approach to doing this.

Base Navigation bottom completed. example is taken from Volkswagen project. Some changes like background, opacity, text color, link and routing edited. Related icons added to assets folder.

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.

I'll provide test steps after finishing settings page

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.
Upcoming and Inprogress are active because their route names are not created yet.
image

image

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

**Comment by atharva3010** [comment]
The link colours are different from the design, below is a screenshot from [Figma](https://www.figma.com/file/Q671pbdFk3omco20xmxk5o/foodouken.com?node-id=2887%3A1).

image

File: src/components/BaseNavigationBottom.vue [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+}
+
+.router-normal-link {
+  @apply opacity-54;
+}
+
+.router-normal-link svg path {
+  fill: #fff;
+  fill-opacity: 0.54;
+}
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:197]: @apply here as well

File: src/components/BaseNavigationBottom.vue [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  transition-duration: 450ms;
+  color: #fff;
+  top: -70px;
+}
+
+ul.toggle-menu.opened > li:nth-child(2) {
+  color: #fff;
+  transition-duration: 450ms;
+  top: -130px;
+}
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:188]: Please use `@apply` if we can, here.

File: src/components/BaseNavigationBottom.vue [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+ul.toggle-menu > li {
+  transition: all 0.2s ease;
+  color: transparent;
+}
+
+ul.toggle-menu.opened > li:nth-child(1) {
+  transition-duration: 450ms;
+  color: #fff;
+  top: -70px;
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:181]: `@apply` here, if we can

File: src/components/LayoutFixedScrollable.vue [hunk 4]
```diff
@@ -14,6 +14,12 @@
...
-  name: 'LayoutFixedScrollable'
+  name: 'LayoutFixedScrollable',
+  data: () => ({
+    isVisible: false
+  }),
+  mounted() {
+    this.isVisible = true;
+  }
 };
 </script>
```
Comment (atharva3010) [inline src/components/LayoutFixedScrollable.vue:25]: I don't think we need LayoutFixedScrollable anymore. BaseAppBarHeader should be set in the root view for tenants (right now it's in Menu.vue) and we control the data passed into it, via meta properties of each route.

Comment (atharva3010) [reply src/components/LayoutFixedScrollable.vue:25]: The Base Bottom Nav can be, used in a similar way.

File: src/components/BaseNavigationBottom.vue (hunk 1 above)
Comment (abdullahcalisir12) [reply src/components/BaseNavigationBottom.vue:197]: I tried to use @apply here but it is not affected as I want

File: src/components/BaseNavigationBottom.vue (hunk 2 above)
Comment (abdullahcalisir12) [reply src/components/BaseNavigationBottom.vue:188]: same here it behaves differently when I use @apply

File: src/components/BaseNavigationBottom.vue (hunk 3 above)
Comment (abdullahcalisir12) [reply src/components/BaseNavigationBottom.vue:181]: same here

**Comment by abdullahcalisir12** [comment]
using **@apply**  causes problem in this case not sure why so I revert it.

**Pull request merged**